src/js/view/utils/add-click-action.ts

```diff
@@ -1,7 +1,7 @@
 import UI from 'utils/ui';
 
-export function addClickAction(element: HTMLElement, clickAction: (evt: Event) => void, ctx?: any): UI {
-    const ui = new UI(element).on('click', clickAction, ctx);
+export function addClickAction(element: HTMLElement, clickAction: (evt: Event) => void, ctx?: any, directSelect?: boolean): UI {
```

Comment:
I don't see a forth `directSelect` option being passed in anywhere. Can you point out where this is being used. Otherwise I don't see the justification for adding it.

Reply:
Removed. I was going to use this for volumeTooltipIcon/future use, but it conflicted with the enter handling since it's a group element.